modules/broker-amqp/src/main/java/io/ballerina/messaging/broker/amqp/Server.java

```diff
@@ -110,6 +114,14 @@ public Server(StartupContext startupContext) throws Exception {
         if (broker == null) {
             throw new RuntimeException("Could not find the broker class to initialize AMQP server");
         }
+
+        if (startupContext.getService(AuthManager.class).isAuthenticationEnabled() &&
```

Comment:
Shall we assign `startupContext.getService(AuthManager.class)` to a local variable? We need to check for null as well since the broker should be able to operate without an auth manager.

Reply:
Fixed in 885868b